docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings.md

```diff
@@ -54,20 +62,27 @@ For instance, the sample JSON snippet below is a tree with nested resources to r
         },
    ...
 ```
+ Within the **Coiote DM - Azure IoT Central integration**, readable and writable resources are interpreted as Properties. While in Azure IoT Hub the data structure is nested, in Azure IoT Central it is represented in one line where objects, object instances, and resources are separated by `n`. For example: `n1n1n1`.
```

Comment:
dałbym tu np n1n0n3 z wyjaśnieneim, że 1 to objec, 0 to obj instance,a 3 to reource